tour_model_eval/run_time_all_data.ipynb

```diff
@@ -0,0 +1,129 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "banned-speaking",
+   "metadata": {},
+   "source": [
+    "## Test run time for bins and clusters on all data"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "sticky-morocco",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import matplotlib\n",
+    "import matplotlib.pyplot as plt\n",
+    "import numpy\n",
+    "import emission.core.get_database as edb\n",
+    "import logging\n",
+    "import folium\n",
+    "import math\n",
+    "import branca.colormap as cm\n",
+    "import emission.storage.timeseries.abstract_timeseries as esta\n",
+    "import emission.analysis.modelling.tour_model.cluster_pipeline as eamtc\n",
+    "import emission.analysis.modelling.tour_model.similarity as similarity\n",
+    "import emission.analysis.modelling.tour_model.cluster_pipeline as pipeline\n",
+    "import emission.analysis.modelling.tour_model.featurization as featurization\n",
+    "import viz_bin_cluster as viz"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "lined-incidence",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "participant_uuid_obj = list(edb.get_profile_db().find({\"install_group\": \"participant\"}, {\"user_id\": 1, \"_id\": 0}))\n",
+    "all_users = [u[\"user_id\"] for u in participant_uuid_obj]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "lyric-evans",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "radius = 300"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "coastal-gabriel",
+   "metadata": {},
+   "source": [
+    "### Run time for bins"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "pressed-stanley",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "for i in range (len(all_users)):\n",
+    "    if i == 11:\n",
+    "        continue\n",
+    "    user = all_users[i]\n",
+    "    trips = pipeline.read_data(uuid=user)\n",
+    "    sim = similarity.similarity(trips, radius)\n",
+    "    %timeit sim.bin_data()"
```

Comment:
where are you saving this output for combing with other results later?

Reply:
no, I didn't. Can't we just copy and paste the result? Or should I calculate the mean runtime?

Reply:
we should not copy-and-paste the result. We should save the result to a machine-readable file. I thought we discussed this today - I distinctly remember bringing up `to_json`, `to_csv` etc.

Reply:
you could also use timeit from python instead if ipython if that is easier to use, and convert the new files to `.py` files

Reply:
@corinne-hcr can you maybe remove the timeit notebooks from this PR so I can merge it and move on to helping you with https://github.com/e-mission/e-mission-eval-private-data/pull/18